packages/quantic/force-app/main/default/lwc/quanticRecommendationList/e2e/pageObject.ts

```diff
@@ -0,0 +1,52 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaClickEvent} from '../../../../../../playwright/utils/requests';
+
+export class RecommendationListObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get recommendationList(): Locator {
+    return this.page.locator('c-quantic-recommendation-list');
+  }
+
+  get recommendationLinks(): Locator {
+    return this.page.locator('c-quantic-result-link a');
+  }
+
+  getRecommendationLinkByIndex(index: number): Locator {
+    return this.recommendationLinks.nth(index);
+  }
+
+  async captureRecommendationListClickEventWorkaround(): Promise<void> {
+    // Hack(?) to have the request payload in the analytics click event.
+    // Without this the payload is not available in the request.
+    return this.page.route('*analytics*', (route) => {
+      route.continue();
+    });
+  }
```

Comment:
@mmitiche we should probably include this somewhere in the Analytics fixture you're making for EP lol